airgun/widgets.py

```diff
@@ -1062,6 +1063,16 @@ def read(self):
         do_not_read_this_widget()
 
 
+class ConfirmationBox(ConfirmationDialog):
+    """PF4 confirmation dialog with two buttons and close 'x' button in the
+    right corner."""
+
+    ROOT = '//div[@id="app-confirm-modal" or @data-ouia-component-type="PF4/ModalContent"]'
+    confirm_dialog = PF4Button('OUIA-Generated-Button-danger-1')
```

Comment:
Wonder why the identifier of confirm button has `danger` word in it, I hope you didnt mismatch the variables here for different dialogs.

Reply:
nah, it's really like that :D